include/oneapi/dpl/internal/common_config.h

```diff
@@ -16,6 +16,10 @@
 #ifndef _ONEDPL_COMMON_CONFIG_H
 #define _ONEDPL_COMMON_CONFIG_H
 
+#if __has_include(<version>)
+#    include <version> // for _GLIBCXX_RELEASE
```

Comment:
[`<version>`](https://github.com/gcc-mirror/gcc/blob/13c83c4cc679ad5383ed57f359e53e8d518b7842/libstdc%2B%2B-v3/include/std/version#L35) from GCC 9 includes [`<bits/c++config>`](https://github.com/gcc-mirror/gcc/blob/13c83c4cc679ad5383ed57f359e53e8d518b7842/libstdc%2B%2B-v3/include/bits/c%2B%2Bconfig#L683) which sets flags like `_GLIBCXX_USE_TBB_PAR_BACKEND`. Perhaps, this should be included after the workarounds below.